mpy-cross/mpy_cross/__init__.py

```diff
@@ -54,17 +55,40 @@
     NATIVE_ARCH_XTENSAWIN,
 ]
 
-__all__ = ["compile", "run", "CrossCompileError"]
+__all__ = ["version", "compile", "run", "CrossCompileError"]
 
 
 class CrossCompileError(Exception):
     pass
 
 
-def find_mpy_cross_binary(mpy_cross):
+def version(march=None, mpy_cross=None):
```

Comment:
Might be better to call this `abi_version` to not confuse it with the mpy-cross/MicroPython version.

Reply:
Also, a doc string explaining the parameters and return value would be helpful.